src/domain/useCases/project/ExportRewards.ts

```diff
@@ -0,0 +1,136 @@
+import {inject, injectable} from 'inversify';
+import {ProjectState} from '../../states/ProjectState';
+import {SchemaFile} from "../../states/objects/fileTree/SchemaFile";
+import {FilesystemId, FilesystemInterface} from '../../context/fs/FilesystemInterface';
+import {SchemaFileVariant} from "../../states/objects/fileTree/SchemaFileVariant";
+import {json2csv} from "json-2-csv";
+
+interface RewardData {
+    id: string;
+    name: string;
+    rewardType: string;
+    featureFlag: string;
+    premiumEquivalent: string;
+    premiumEquivalentExchange: string;
+    numberOfFragmentsNeeded: string;
+}
+
+const createNewRewardData = (): RewardData => {
```

Comment:
Why didn't you put `createNewRewardData` inside of the `ExportRewards` class and made it private? 

It seems that it is not used anywhere else.